packages/console/src/pages/SignInExperience/PageContent/SignUpAndSignInChangePreview/index.tsx

```diff
@@ -9,14 +11,29 @@ import styles from './index.module.scss';
 type Props = {
   readonly before: SignInExperiencePageManagedData;
   readonly after: SignInExperiencePageManagedData;
+  readonly isForgotPasswordMigrationNoticeVisible: boolean;
 };
 
-function SignUpAndSignInChangePreview({ before, after }: Props) {
+function SignUpAndSignInChangePreview({
+  before,
+  after,
+  isForgotPasswordMigrationNoticeVisible,
```

Comment:
Adding a `isForgotPasswordMigrationNoticeVisible` prop to `SignUpAndSignInChangePreview` component seems unrelevant to me. It would be better if you change the prop to a conditional `readonly notification?: ReactNode;`.

Reply:
But this isn’t a general-purpose component and there aren’t other use cases. I think we can just wait and refactor later if needed.